DigitalLearningSolutions.Web/Views/Support/SupportTickets.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.Support
+@model SupportTicketsViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/support/support.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Support tickets";
+  ViewData["Application"] = Model.DlsSubApplication.HeaderExtension;
+  ViewData["HeaderPath"] = Model.DlsSubApplication.HeaderPath;
+  ViewData["HeaderPathName"] = Model.DlsSubApplication.HeaderPathName;
+}
+
+@section NavMenuItems {
+  @if (Model.DlsSubApplication.Equals(DlsSubApplication.TrackingSystem)) {
+    <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+  } else if (Model.DlsSubApplication.Equals(DlsSubApplication.Frameworks)) {
+    <partial name="~/Views/Frameworks/Shared/_NavMenuItems.cshtml" />
+  }

```

Comment:
Sorry, actually looks like this is being dealt with in 608, so never mind about this.

Reply:
Either this needs dealing with here or a todo adding (and a note on the 608 ticket). I see no reason why it can't be dealt with here, it was possible for the ResourcesController/View from 521. Saves a bit of work on 608.

Reply:
Ah yes, done.